src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -51,7 +51,7 @@ private void ReloadFile(FileReload fileReload)
 
             if (!fileReload.IsValid())
             {
-                if (fileReload.FilePath.HasValue() && this.Log().IsEnabled(LogLevel.Debug))
+                if (fileReload.FilePath.IsNullOrEmpty() && this.Log().IsEnabled(LogLevel.Debug))
```

Comment:
The change from `HasValue` to `IsNullOrEmpty` is intentional. I think the old code is wrong.